kube/helm/app/templates/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: db-secret
+  namespace: vegait-training
+type: Opaque
+data:
+  POSTGRES_PASSWORD: {{ .Values.postgresql.password | b64enc }}
```

Comment:
Why do you do base64 ?

Reply:
Without that, I had a problem in that my app failed to authenticate to db and did not make the password visible as plain text(a bit obfuscating).

Reply:
Something else was the issue there, you don't need it as each secret by default ends up base64 encoded by K8S.